packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/WPAndroidGlue/DeferredEventEmitter.java

```diff
@@ -226,4 +229,10 @@ public void sendToJSFeaturedImageId(int mediaId) {
     public void updateCapabilities(GutenbergProps gutenbergProps) {
         queueActionToJS(MAP_KEY_UPDATE_CAPABILITIES, Arguments.makeNativeMap(gutenbergProps.getUpdatedCapabilitiesProps()));
     }
+
+    public void updateEditorSettings(String rawEditorSettings) {
```

Comment:
This event has the same requirements as `updateTheme` [1](https://github.com/WordPress/gutenberg/pull/33014/files#diff-26b36175b00a8403113cae205e5b3c1819c3206e2c8b5baaebb148a1a61581bcR681). I think we should work to consolidate those events so that we don't cause weird or unexpected race conditions based on when they fire and trigger rerenders.